site/plans.html.jet

```diff
@@ -18,35 +11,57 @@
       <h1>{{ i18n("plans_page_header") }}</h1>
     </div>
 
-    <div class="container">
+    <div class="container-xxxl px-2 px-lg-5">
       <div class="row">
       {{ range _, p := site.Plans }}
-        <div class="plan col-md-4">
-          {{ if isset(p.PortraitImage)}}
-            <s72-image src="{{p.PortraitImage}}" alt="{{p.Name}}" class="plan-poster card-img-top" border="0"></s72-image>
-          {{ end }}
-            <div class="plan-caption">
-              <div class="plan-name">{{p.Name}}</div>
-              {{ if p.HasExpiryDate() }}
-                <div class="plan-expiry">{{ i18n("plan_expiry_date", map("ExpiryDate", p.ExpiryDate)) }}</div>
-              {{ end }}
-              <div class="plan-description">{{p.Description}}</div>
-            </div>
-
-          {{ if p.Interval }}
-            <div class="plan-details">
-              <div class="plan-interval">{{i18n("shopping_info_plan_offer", map("Interval", p.Interval, "Count", p.IntervalCount))}}</div>
-              {{ if p.TrialPeriodDays }}
-                <div class="plan-trial-period">{{i18n("shopping_info_trial_period_offer", p.TrialPeriodDays)}}</div>
-              {{ end }}
-            </div>
-          {{ end }}
-          <div class="plan-buttons">
-            {{ if p.Page }}
-              <button class="plan-page-link"><a href="{{ routeToSlug(p.Page.Slug) }}">{{i18n("plan_read_more")}}</a></button>
+        <div class="plan col-xs-6 col-lg-4 mb-3">
+          <div class="card h-100">
+            {{ if isset(p.LandscapeImage) && p.LandscapeImage != ""}}
```

Comment:
A minor nit: the majority of the templates have no whitespace after/before the mustaches.

Reply:
- removed space before/after moustaches